internal/util/parse.go

```diff
@@ -110,3 +112,20 @@ func ParseBool(b string) *bool {
 	}
 	return &truth
 }
+
+// Transforms uint32 to ipv4 representation
+func uint322Ipv4(n uint32) string {
```

Comment:
let's rename this to uint32ToIPv4.
You can also probably shorten the body to `return binary.LittleEndian.PutUint32(net.IP{}, uint32(n)).String()`

Reply:
seems not possible to shorten its body like your said way, happy to rename though.